deepseg.cc

```diff
@@ -148,8 +148,23 @@ typedef struct {
 	pthread_mutex_t lock;
 } capinfo_t;
 
+enum class modeltype {
```

Comment:
name consistency -> `modeltype_t` ?

Reply:
Wanted to keep it clean without _t since type name is used as prefix on each use. I'll add _t suffix.